### PR TITLE
Sage/how to build a basic crud app in solidity

### PR DESCRIPTION
**Pull request by iamoracle**
All the issues with the former PR has been addressed

### CODE REVIEW

**Comment by yafiabiyyu** [comment]
I have finished reviewing this article, just waiting for you to approve it @joenyzio @viral-sangani

**Comment by yafiabiyyu** [comment]
Alterations that ought to be executed upon your pull request :
1. Relocate the folder `01-21-2023-how-to-build-a-basic-crud-app-in-solidity`
 into the `blog/` folder."
 
2. Incorporate the following code on the first line of the tutorial file and populate the required data.
```markdown
---
title:
description:
authors:
  - name: 
tags:
hide_table_of_contents: true
slug: "/tutorials/how-to-build-a-basic-crud-app-in-solidity"
---
```

3. For the header image, you may retrieve it from your Trello card and save it within the `src/data-tutorials/showcase/beginner` folder. I recommend naming the file according to the title of the tutorial you created.

4. To have your tutorial added to the list, you will need to edit the file `src/data-tutorials/users.tsx` and edit the code below. You can follow the example that already exists in the `users.tsx`
 file.
 ```markdown
 {
	title: '',
	description: '',
	preview: require(''),
	website: '',
	tags: ['beginner', celosage],
}
```

5. finally, ensure you run prettier for your markdown file.

Should you still struggle to grasp what has already been explained, you may pursue two videos provided by the Celo team
- [How to create your first tutorial](https://www.loom.com/share/857c464d0ccf45b8879fb7fae9f291ba)
- [How to create a PR for your tutorial](https://www.loom.com/share/10c6437eee5a44c782692c3b59e58b71)

**Comment by iamoracle** [comment]
All issue has been addressed. Kindly merge.

File: blog/01-21-2023-how-to-build-a-basic-crud-app-in-solidity/how-to-build-a-basic-crud-app-in-solidity.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
+---
+title: How to build a Basic CRUD App in Solidity
+description: This post will teach you how to create a basic Create, Read, Update and Delete (CRUD) smart contract and test it with the Truffle framework.
+authors:
+  - name: Oyeniyi Abiola Peace
+    title: Community Moderator, Celo
+    url: https://github.com/iamoracle
+    image_url: https://github.com/iamoracle.png
+tags: [CRUD, Celo Sage, Solidity]
```
Comment (yafiabiyyu) [inline blog/01-21-2023-how-to-build-a-basic-crud-app-in-solidity/how-to-build-a-basic-crud-app-in-solidity.md:9]: For tags, please refer to the file `src/data-tutorials/users.tsx` in the `TagType` section

File: src/data-tutorials/users.tsx [hunk 2]
```diff
@@ -831,6 +831,13 @@ const Users: User[] = [
     website: 'blog/tutorials/flutter-celo-easily-build-flutter-mobile-dApps',
     tags: ['intermediate','foundation', 'flutter', 'composer'],
   },
+  {
+    title: 'How to build a Basic CRUD App in Solidity',
+    description: 'This post will teach you how to create a basic Create, Read, Update and Delete (CRUD) smart contract and test it with the Truffle framework.',
+    preview: require('./showcase/celo-sage.png'),
```
Comment (yafiabiyyu) [inline src/data-tutorials/users.tsx:837]: For the showcase image you can get it on your trello card, save it in the `src/data-tutorials/showcase/{beginner/intermediate/advandced} folder

File: blog/01-21-2023-how-to-build-a-basic-crud-app-in-solidity/how-to-build-a-basic-crud-app-in-solidity.md [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+hide_table_of_contents: true
+slug: /tutorials/how-to-build-a-basic-crud-app-in-solidity
+---
+
```
Comment (yafiabiyyu) [inline blog/01-21-2023-how-to-build-a-basic-crud-app-in-solidity/how-to-build-a-basic-crud-app-in-solidity.md:13]: Add image headers
`header`

**Pull request closed without merging**